### DISCUSSION TITLE
Unsupported Unicode Chars

### DISCUSSION THREAD

**Discussion by SeSe008** [emilk/egui#3743] [Q&A]
In an egui application, i want select a random char out of all of Unicode (`random_scalar_value = rng.gen_range(0x0000..=0xD7FF);
random_char = std::char::from_u32(random_scalar_value).unwrap();`) and show it on screen.
My problem is, that eguii doesnt support most of the chars, as egui has a limited font atlas.
Is there any workaround to render these chars?

**Comment by emilk** [comment]
Yes - install a font that supports them using https://docs.rs/egui/latest/egui/struct.Context.html#method.set_fonts
